keras_hub/src/models/retinanet/retinanet_label_encoder_test.py

```diff
@@ -1,26 +1,33 @@
 import numpy as np
 from keras import ops
 
+from keras_hub.src.models.retinanet.anchor_generator import AnchorGenerator
 from keras_hub.src.models.retinanet.retinanet_label_encoder import (
     RetinaNetLabelEncoder,
 )
 from keras_hub.src.tests.test_case import TestCase
 
 
 class RetinaNetLabelEncoderTest(TestCase):
+    def setUp(self):
+        self.anchor_generator = AnchorGenerator(
+            bounding_box_format="xyxy",
+            min_level=3,
+            max_level=7,
+            num_scales=3,
+            aspect_ratios=[0.5, 1.0, 2.0],
+            anchor_size=8,
+        )
+
     def test_layer_behaviors(self):
         images_shape = (8, 128, 128, 3)
         boxes_shape = (8, 10, 4)
         classes_shape = (8, 10)
         self.run_layer_test(
             cls=RetinaNetLabelEncoder,
             init_kwargs={
+                "anchor_generator": self.anchor_generator,
                 "bounding_box_format": "xyxy",
-                "min_level": 3,
-                "max_level": 7,
-                "num_scales": 3,
-                "aspect_ratios": [0.5, 1.0, 2.0],
-                "anchor_size": 8,
             },
             input_data={
```

Comment:
define input_data also in setup()

Reply:
input_data varies in each test_case so kept it separately also how we assert changes